Documentation/Contributing/development-flow.md

```diff
@@ -243,6 +248,28 @@ Common cases that may need tests:
 * an input is specified incorrectly, for each input
 * a resource the code relies on doesn't exist, for each dependency
 
+## Linting
+
+The CI will run various linters against your changes to check code, manifests, and docs for syntactic correctness.
+The various linters can be run locally as follows.
+
+Run `make lint` to run all linters. This will require the following CLI tools to be installed on the system:
+
+* [yamllint](https://www.yamllint.com/)
+* [shellcheck](https://www.shellcheck.net/)
+* [pylint](https://www.pylint.org/)
+* [markdownlint-cli2](https://github.com/DavidAnson/markdownlint-cli2)
+* [go vet](https://pkg.go.dev/cmd/vet) 
+* [golangci-lint](https://golangci-lint.run/)
+
+The exception to this requirement of local installation  is golangci-lint.
```

Comment:
The below statement is unnecessary if you remove golangci-lint from the bulleted list, so please remove the bullet entry

Reply:
done